presidio-api/cmd/presidio-api/docs/docs.go

```diff
@@ -0,0 +1,16 @@
+// Package classification Presidio.
+//
+// Documentation for Presidio API Swagger.
+//
+//     Schemes: http, https
+//     BasePath: /api/v1
+//     Version: 1.0
+//
+//     Consumes:
+//     - application/json
+//
+//     Produces:
+//     - application/json
+//
+// swagger:meta
+package docs
```

Comment:
do we must have these code files inside a doc directory?

Reply:
yes, this is the meta-data about the swagger, and it is referenced by the main.go to let the go-swagger find the docs.